Reworked Rage Mechanic (#132)
Jihia/Warcraft-Heroes-Handbook: Heroes Handbook, Main File.txt

You can spend 2 rage points before making an attack to attempt to disarm the target. If you hit, the target loses the grip on their weapon, the weapon lands in an empty square of your choice within 10 feet of the target.

#### Execute
When you hit with a weapon attack, you can spend 6 rage points to ferociously wound the target. When done, you deal extra weapon damage to the target equal to your Warrior level.

#### Hamstring

> You can spend 3 rage points before making an attack to attempt to hinder the targets movement. If your attack hit, the target must make a Constitution saving throw, having their speed reduced to 0 on a fail, or half on a successful save until the beginning of your next turn.

Reviewer: ```suggestion
You can spend 3 rage points before making an attack to attempt to hinder the target's movement. If your attack hits, the target must make a Constitution saving throw. The target's speed is reduced to 0 on a failed save, or half on a successful one, until the start of your next turn.
```

Once you use this feature, you must finish a short or long rest before you can use it again. Starting at 17th level, you can use it twice before a rest, but only once per turn.

### Dauntless
By 10th level, your instincts are so honed that you have advantage on initiative rolls.

> In addition, if you are surprised at the beginning of combat and aren't incapacitated, you can use your first turn in combat to enter a rage and nullify the surprise condition.

Reviewer: ```suggestion
In addition, if you are surprised at the beginning of combat, you can act normally on your first turn if you choose to enter a rage as part of that turn.
```

You can use this feature three times, and you regain all expended uses of it when you finish a long rest.

#### Titan's Grip

> Starting at 7th level, you can wield weapons that has the *two-handed* melee weapon property with one hand, gaining all of the benefits that otherwise comes with the weapon. To do this you must be proficient in the weapons use.

Reviewer: ```suggestion
Starting at 7th level, you can wield melee weapons that have the *two-handed* property with one hand, gaining all of the benefits that otherwise comes with the weapon. To do this you must be proficient in the weapons use.
```
